packages/fuels-abigen-macro/tests/test_projects/contract_output_test/Forc.lock

```diff
@@ -2,7 +2,7 @@
 name = 'contract_output_test'
```

Comment:
This file should be removed? unless it's special

Reply:
Oh no, it should!